lib/tasks/docs.thor

```diff
@@ -65,8 +65,7 @@ class DocsCLI < Thor
     handle_doc_not_found_error(error)
   end
 
-  desc 'generate <doc> [--version] [--verbose] [--debug] [--force] [--package]', 'Generate a documentation'
-  option :version, type: :string
+  desc 'generate <doc> <doc@version>... [--verbose] [--debug] [--force] [--package]', 'Generate a documentation'
```

Comment:
```suggestion
  desc 'generate (<doc> <doc@version>) [--verbose] [--debug] [--force] [--package]', 'Generate a documentation'
```

As running generate on two docs does not work:
```
bundle exec thor docs:generate --force --package vue@2 vue@3
ERROR: "thor generate" was called with arguments ["vue@2", "vue@3"]
Usage: "thor docs:generate <doc> <doc@version>... [--verbose] [--debug] [--force] [--package]"
```

Reply:
Sorry I missed this! I must have glanced at `download` usage and copied that.

I would write a required single value with optional syntax as: `(<doc> | <doc@version>)`; using `|` as "or", consistent with the doc sets in the description of `docs:download`.  Space separated with parens still looks, to me, like more than one "docs" could be specified.

Let me know if you'd prefer your suggestion.

You mentioned two issues-- what is the other?

Reply:
> Please also update CONTRIBUTING, section "Updating existing documentations": merge step 7 into step 5?

I hadn't noticed this file, sorry--  now done.  Hopefully with the new version notation, how to repeat in step 7 is evident.